easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb

```diff
@@ -18,7 +18,9 @@ postinstallcmds = [('ln -s %(installdir)s/lib64 %(installdir)s/lib')]
 
 builddependencies = [
     ('CMake', '3.7.2', '', True),
-    # ('Doxygen', '1.8.9.1'),
+]
+
+dependencies = [
     ('cURL', '7.52.1'),
```

Comment:
Is it still needed ?
https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-gmvolf-17.02.eb#L22

Reply:
yes

Reply:
I meant `curl` is already a `builddependencies` , see the link to master.
Is it needed as a `dependencies` ?

Reply:
See http://www.unidata.ucar.edu/software/netcdf/docs/getting_and_building_netcdf.html#build_default and this old post as well: https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg09780.html